### ISSUE TITLE
LV2 worker thread produce Xruns

### ISSUE THREAD

**Issue by brummer10** [falkTX/Carla#1300]
As reported in IRC, the LV2 worker thread in carla seems to have a significant issue. 
When it get work to do, like load a IR-file with LSP convolvers, or, load a sf2 file with a fluidsynth wrapper, Xruns happen.
This doesn't happen in other host's like Ardour, Qtractor or jalv. As well that doesn't happen when the LV2 plugin is wrapped with lv2vst as VST plugin. 

I just checked the CPU usage of the carla threads and noticed that the CPU usage from the rt thread of the plugin which span the worker thread, increase when the worker work.

**Comment by brummer10** [comment]
Doing some debugging, check the thread pid of the engine and the worker thread:
In engine I do
`std::cout <<  "engine " <<std::this_thread::get_id() << std::endl;`
and in worker I do
`std::cout <<  "work " <<std::this_thread::get_id() << std::endl;`

result in Carla:

```
engine 139834878084864
work 139834878084864
```

as you could see, that is the same thread. Means work is done in the dsp thread.

any other host gives results like:

```
engine 140328732944128
work 140328570480384
```

**Comment by falkTX** [comment]
Fixed this a while ago in the 2.3 series, just forgot to close.
